torch/csrc/jit/python/init.cpp

```diff
@@ -1308,18 +1358,21 @@ void initJITBindings(PyObject* module) {
             });
       });
 
-  m.def("_jit_assert_is_instance", [](py::object obj, const TypePtr& type) {
+  m.def("assert_is_instance", [](py::object obj, const TypePtr& type) {
     toIValue(std::move(obj), type);
   });
 
-  initPythonCustomClassBindings(module);
-  initPythonIRBindings(module);
-  tracer::initPythonTracerBindings(module);
-  initTreeViewBindings(module);
-  initJitScriptBindings(module);
-  initJitBackendBindings(module);
-  initStaticRuntimeBindings(module);
-  initTensorExprBindings(module);
+  initPythonCustomClassBindings(m.ptr());
+  initPythonIRBindings(m.ptr());
+  tracer::initPythonTracerBindings(m.ptr());
+  top.def(
```

Comment:
nit: do we want to add a comment here with the issue as well? Explaining why this was moved outside of the tracer init?

Reply:
I added a comment here too. This one is actually fixable because the use of `torch._C._get_tracing_state` that we need to fix is in torchvision.